build.gradle

```diff
@@ -105,4 +108,39 @@ subprojects {
     dependencies {
         testImplementation 'ch.qos.logback:logback-classic:1.2.3'
     }
+
+    spotless {
+        enforceCheck false
+        java {
+            removeUnusedImports()
+            trimTrailingWhitespace()
+            endWithNewline()
+
+            prettier(['prettier': '2.5.1', 'prettier-plugin-java': '1.6.1'])
+                .config([
+                    'parser'    : 'java',
+                    'tabWidth'  : 4,
+                    'printWidth': 120
+                ])
+
+            importOrder('', 'java', 'javax', '\\#')
+        }
+        groovyGradle {
+            target '**/*.groovy'
+            greclipse('4.19.0')
+        }
+    }
+
+    checkstyle {
+        toolVersion = "9.3"
+        configFile = rootProject.file('config/checkstyle/checkstyle.xml')
+        checkstyleTest.enabled = false
+        checkstyleMain.enabled = false
+    }
+
+    tasks.whenTaskAdded {task ->
+        if (task.name.equals("checkstyleJarFileTest")) {
+            task.enabled = false
+        }
+    }
```

Comment:
I think you can try this:
```groovy
tasks.withType(Checkstyle.class)*.enabled = false
```

You _may_ need to wrap it with `project.afterEvaluate` tho, but probably not

Reply:
Another option would be to set `ignoreFailures = true` on `checkstyle`, btw